initfiles/sbin/hpcc/cluster/task.py

```diff
@@ -131,7 +131,7 @@ def worker(self, host):
         protect the script from malicious modification.
         '''
         try:
-            self._ip = host.ip.decode('utf-8')
+            self._ip = host.ip
```

Comment:
@xwang2713 I believe this issue is already properly fixed in 8.0.x.  If we hit an AttributeError here from the decode('utf-8') being present on a string object (which there isn't a guarantee that host.ip actually is a string object because of how python3 does dynamic casting), then we'll fall through the try block and assign host.ip straight to self._ip like you  have on line 134, and hpcc-run.sh will work fine.  I've tested this out on the 160 last week.  Perhaps whoever raised this bug was not using the latest 8.0.x build?

Reply:
Do you know which 8.0.x?  I built AMI with 8.0.2-1 but hpcc-run.sh throws an exception for host.ip.decode()

Reply:
Previously we use Python2.7 which doesn't have this problem. Probably Python2.7 ignore the case.

Reply:
Earliest branch that I see the commit in is 8.0.4 (4b28584532aac1403ebbdc9e7e94275712faced3)

Python2 didn't have this issue because decode/encode were always available to string objects. They weren't unicode by default, so there was never an attributeError to throw.

Reply:
OK. Ignore this PR